src/structures/Guild.js

```diff
@@ -175,6 +175,12 @@ class Guild extends Base {
      */
     this.joinedTimestamp = data.joined_at ? new Date(data.joined_at).getTime() : this.joinedTimestamp;
 
+    /**
+     * The Shard ID that this guild is on
+     * @type {number}
+     */
+    this.shardId = data.shard ? data.shard.id : null;
```

Comment:
Should the default ID be 0? That way, the `@type` will always match and it makes sense in the long run